tests/results/DiffTest.py

```diff
@@ -30,6 +30,11 @@ def test_delete_line(self):
         self.uut.delete_line(1)  # Double deletion possible without conflict
         self.assertRaises(ValueError, self.uut.delete_line, 0)
 
+    def test_delete_lines(self):
+        self.uut.delete_lines(1, 10)
+        self.uut.delete_lines(10, 20)
+        self.assertRaises(ValueError, self.uut.delete_lines, 0, 10)
```

Comment:
ahm so this will only raise a valueerror because you're trying to delete line 0, you're not really testing if th line deletion is actually happening - OTOH the other tests aren't really better. Maybe I wrote them... :P

Could you at least test the `len` of uut or something to assure that the deletions happen?